hs/ConvertSVGToPNG.hs

```diff
@@ -0,0 +1,48 @@
+module ConvertSVGToPNG where
+
+import System.Process
+import GHC.IO.Handle.Types
+import Control.Monad.IO.Class  (liftIO)
+import GHC.IO.Exception
+
+-- | Converts an SVG file to a PNG file. Note that image magik's 'convert' command
+-- can take in file descriptors.
+convertSVGToPNG :: String -> String -> IO
+                     (Maybe Handle,
+                      Maybe Handle,
+                      Maybe Handle,
+                      ProcessHandle)
+
+convertSVGToPNG inName outName = createProcess $ CreateProcess
+                                  (ShellCommand $ "convert ../res/graphs/graph_regions.svg INSERT_ID-graph.png"
+                                   )
+                                  Nothing
+                                  Nothing
+                                  CreatePipe
+                                  CreatePipe
+                                  CreatePipe
+                                  False
+                                  False
```

Comment:
The library has changed since I first installed it. My version only requires 2, the newest version requires 3. Are we keeping up to date with System.Process?